src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -15,6 +15,7 @@
 import gov.usgs.wma.waterdata.collections.geojson.CollectionGeoJSON;
 import gov.usgs.wma.waterdata.collections.geojson.CollectionsGeoJSON;
 import gov.usgs.wma.waterdata.collections.geojson.FeatureGeoJSON;
+import gov.usgs.wma.waterdata.collections.observations.ObservationsJSON;
```

Comment:
This controller is starting to get long. Might be a good ideal to split it up (like the integration tests are for it) if any more endpoints are added.

Reply:
agreed, we are going to have at least one other endpoint under observation and likely
additional observation types other than statistical-time-series.
So may as well start with ObservationsController

Reply:
I would not bother me to have a class per endpoint.